docs/hyper-params.ipynb

```diff
@@ -24,16 +24,17 @@
     "\n",
     "The hyperparameters and options are specified in the `task` or the {py:meth}`~mlrun.runtimes.BaseRuntime.run` command \n",
     "through the `hyperparams` (for hyperparam values) and `hyper_param_options` (for \n",
-    "{py:class}`~mlrun.model.HyperParamOptions`) properties. See the examples below. Hyperparameters can also be loaded directly \n",
-    "from a CSV or Json file (by setting the `param_file` hyper option).\n",
+    "{py:class}`~mlrun.model.HyperParamOptions`) properties. See the examples below. \n",
     "\n",
-    "The hyperparams are specified as a struct of `key: list` values for example: `{\"p1\": [1,2,3], \"p2\": [10,20]}`. The values \n",
-    "can be of any type (int, string, float, ..). The lists are used to compute the parameter combinations using one of the \n",
+    "The hyperparams are specified as a struct of `key: list` values. The values can be of any type (int, string, float, ..). \n",
+    "The lists are used to compute the parameter combinations using one of the \n",
     "following strategies: \n",
-    "- [Grid search](#grid-search-default) (`grid`) &mdash; running all the parameter combinations.\n",
-    "- [Random](#random-search) (`random`) &mdash; running a sampled set from all the parameter combinations.\n",
-    "- [List](#list-search) (`list`) &mdash; running the first parameter from each list followed by the seco2nd from each list and so on. **All the lists must be of equal size**.\n",
-    "- [Custom](#custom-iterator) (`custom`) &mdash; determine the parameter combination per run programmatically. \n",
+    "- [Grid search](#grid-search-default) (`grid`) &mdash; running all the parameter combinations. The `key: list` values structure is similar to: \n",
+    "  ` { \"p1\": [1,2], \"p2\": [2,4] }`<br>\n",
+    "   The result is the four iterations with all the combinations of p1 and p2. \n",
+    "   Hyperparameters can also be loaded directly from a JSON file (specify `param_file` in {py:class}`~mlrun.model.HyperParamOptions`).\n",
+    "- [Random](#random-search) (`random`) &mdash; running a sampled set from all the parameter combinations. Hyperparameters can       also be loaded directly from a JSON file, the same as `grid`.\n",
+    "- [List](#list-search) (`list`) &mdash; running the first parameter from each list followed by the second from each list and so on. **All the lists must be of equal length**. Hyperparameters can also be loaded directly from a CSV file containing a list of the iterations to be executed, for example: `{\"p1\": [1], \"p2\": [10]}` (specify `param_file` in {py:class}`~mlrun.model.HyperParamOptions`).\n",
```

Comment:
The example provided is of a JSON file, not of a CSV file.
In this case (and only in this case), the hyperparams can be in either CSV or JSON format.